sapmon/payload/provider/sqlserver.py

```diff
@@ -185,25 +189,84 @@ def generateJsonString(self) -> str:
                                                                                 e))
       return resultJsonString
 
+   # Prepare the SQL statement based on the check-specific query
+   def _prepareSql(self,
+                   sql: str,
+                   isTimeSeries: bool,
+                   initialTimespanSecs: int) -> str:
+      self.tracer.info("[%s] preparing SQL statement" % self.fullName)
+
+      # Insert logic to get server UTC time (_SERVER_UTC)
+      sqlTimestamp = ", GETDATE() AS %s, GETUTCDATE() AS %s, GETUTCDATE() AS %s FROM" % (COL_LOCAL_UTC,COL_SERVER_UTC,COL_TIMESERIES_UTC)
```

Comment:
```suggestion
      sqlTimestamp = ", GETUTCDATE() AS %s, GETUTCDATE() AS %s FROM" % (COL_SERVER_UTC,COL_TIMESERIES_UTC)
```
We only need COL_SERVER_UTC here:
- COL_LOCAL_UTC is the current UTC time of the collector VM (this is fetched via `datetime.utcnow()` in function `updateState()`
- COL_TIMESERIES_UTC is the column in the result set of a time-series query that stores the past UTC time of the particular row